packages/notifi-react-example/src/NotifiCard/NotifiCard.tsx

```diff
@@ -60,6 +60,10 @@ export const NotifiCard: React.FC = () => {
   };
 
   return (
+    /**
+     * Test dappAdresses/ subscription cardId / Intercom cardId:
+     * junitest.xyz / b07f0267df1343db9d14757c653ec092 / 1045f61752b148eabab0403c08cd60b2
```

Comment:
do we need these comments here?

there is a chance we delete these ids in the future

Reply:
Thanks for the head-up. comments removed.